This patch fixes builds on OSes that don't support running Windows/.NET
PE32 `.exe' files, such as GNU/Linux without proper `binfmt_misc' support.

diff --git a/csharp/Makefile.am b/csharp/Makefile.am
index a54b8f2..c73404d 100644
--- a/csharp/Makefile.am
+++ b/csharp/Makefile.am
@@ -73,7 +73,7 @@ GenerateTables.exe: $(SOURCES_GENERATE)
 
 RFC3454.cs CombiningClass.cs DecompositionKeys.cs DecompositionMappings.cs Composition.cs: $(GEN_SOURCES)
 	make GenerateTables.exe
-	./GenerateTables.exe
+	mono ./GenerateTables.exe
 
 install-dll-dir:
 	$(mkinstalldirs) $(DESTDIR)$(libdir)
